src/Sarif.FunctionalTests/SarifConverterTests.cs

```diff
@@ -116,7 +147,7 @@ private void RunConverter(StringBuilder sb, string toolFormat, string inputFileN
             {
                 sb.AppendLine(string.Format(CultureInfo.InvariantCulture, "The converter {0} threw an exception for input \"{1}\".", toolFormat, inputFileName));
                 sb.AppendLine(ex.ToString());
-                return;
+                return generatedFileName;
```

Comment:
>generatedFileName [](start = 23, length = 17)

How about returning `null` on error, and then caller can test for `null` instead of having to test `File.Exists`? I thought that was odd when I saw the test on line 96; I wondered to myself "Why would it not exist?".